tasks/src/text-to-image/data.ts

```diff
@@ -26,7 +26,21 @@ const taskData: TaskDataCustom = {
 			},
 		],
 	},
-	metrics: [],
+	metrics: [
+		{	description: " The Inception Score (IS) measures the quality of a model's image generation by assessing diversity and meaningfulness. It uses a generated image sample x to predict its label y using the Inception model. 
+					The IS is then calculated by measuring the KL divergence between the marginal distribution p(y) and the conditional distribution p(y|x). A higher divergence signifies more diverse and meaningful images, resulting in a higher IS score.",
```

Comment:
```suggestion
.",
```
For these pages, we aim for quite simple language and avoid the jargon, so it would be even better if you could indicate something like: Higher IS score indicates more diverse and meaningful images. Does this work?

Reply:
Yes, I will update this with less details 👍